getting_started/step_by_step/your_first_game.rst

```diff
@@ -206,6 +234,36 @@ or ``false`` if it isn't.
         else:
             $AnimatedSprite.stop()
 
+ .. code-tab:: csharp
+
+    public override void _Process(float delta)
+    {
+        var velocity = new Vector2();
+        if(Input.IsActionPressed("ui_right")) {
```

Comment:
It's trivial, but I guess we used a space between `if` and `(` in other places, so I suppose it'd be better to keep the conding style consistent across the documentation.